app/src/app/views/landing-page/landing-page.component.ts

```diff
@@ -44,6 +44,7 @@ export class LandingPageComponent implements OnInit, OnDestroy {
         if (this.authenticationService.isAuthorized([Role.ADMIN])) {
             let subscription = this.restApiConnector.getOrganizationIdentity().subscribe({
                 next: (identity) => {
+                    // TODO add popup for editing placeholder namespace prefix?
```

Comment:
You can probably just remove this TODO. I think since the namespace configuration is completely optional, we don't need to modify the dialog for the organization identity or add a new one for namespacing. The existing one will take them to the settings page where they can set a namespace if they wish to.